website/content/api-docs/secret/identity/mfa/index.mdx

```diff
@@ -0,0 +1,26 @@
+---
+layout: api
+page_title: /identity/mfa - HTTP API
+description: >-
+  The '/identity/mfa' endpoint focuses on managing login MFA behaviors in Vault.
+---
+
+# `/identity/mfa`
+
+## Supported MFA types.
+
+- [TOTP](/api/secret/identity/mfa/totp)
+
+- [Okta](/api/secret/identity/mfa/okta)
+
+- [Duo](/api/secret/identity/mfa/duo)
+
+- [PingID](/api/secret/identity/mfa/pingid)
+
+## Other
```

Comment:
Instead of other, could this be something more specific, like say `Configuration` ?

Reply:
Well, technically all of this is configuration. I'm definitely open to changing it away from `Other`, but I'm worried that putting `Configuration` will imply that the supported MFA types are _not_ configuration, and that's not really true.

Reply:
ah yes, Configuration is not a great choice then. it's a nit at any rate, if there's not really an appropriate label for the enforcement api i think Other should be totally fine too.